docs/transparency.md

```diff
@@ -0,0 +1,220 @@
+# Transparency
+
+The ArmoredWitness device is built to help improve the security properties of
+ecosystems which rely on [transparency](https://transparency.dev) logs.
+
+It does this by:
+
+* **Observing public transparency logs** \
+  Verifying that they're operating in an append-only fashion, and
+  counter-signing those checkpoints which it has determined are consistent
+  with all previous checkpoints its seen from the same log.
+* **Making these counter-signed checkpoints available** \
+  Sending them to a
+  [distributor](https://github.com/transparency-dev/distributor), which
+  then collates counter-signatures for a given checkpoint from one or more
+  ArmoredWitness devices, and serves them via a public API.
+
+The benefit of this system comes though removing trust from log operators to
+behave honestly, and placing some of that trust in the witnesses. Splitting the
+trust across multiple parties in this way means that a larger number of parties
+must collude to hide malfeasance, and as other witness implementations/networks
+start to appear, the number of parties required to collude increases
+correspondingly.
+
+However, we can minimise the amount of trust required to be placed in the
+ArmoredWitness by having it be as transparent as possible too.
+
+## Minimising trust
+
+In order to minimise the level of trust required to be placed in the ArmoredWitness,
+we have designed & implemented it such that:
+
+* All firmware is opensource, written in [TamaGo](https://github.com/usbarmory/tamago),
+  and is build-reproducible by anyone.
+* All firmware is logged to a Firmware Transparency log at build and release time.
+* The [`provision`](cmd/provision/) tool will only use firmware artefacts discovered
+  in the FT log in order to program devices.
+* The on-device self-update process requires that updated firmware is hosted in the 
+  FT log.
+* The boot "chain of trust" requires valid "off-line FT proof bundles" to be present
```

Comment:
off-line -> offline?

Reply:
I think either is correct, but dropped the hyphen to be hipper :)